mr/src/test/java/org/apache/iceberg/mr/hive/TestTables.java

```diff
@@ -83,12 +92,107 @@ public Tables tables() {
   public abstract String locationForCreateTableSQL(TableIdentifier identifier);
 
   /**
-   * If the {@link Catalogs#LOCATION} is needed for {@link Catalogs#loadTable(Configuration, Properties)} then this
-   * method should provide the location string. It should return <code>null</code> if the location is not needed.
-   * @param identifier The table identifier
-   * @return The location string for loadTable operation
+   * If creating the Hive table independently is needed for the given Catalog then this should return the Hive SQL
+   * string which is needed to be executed.
+   * @param identifier The table identifier (the namespace should be "default")
+   * @return The SQL string - which should be executed, null - if it is not needed.
```

Comment:
it doesn't seem like it will give back null in any scenario - when should it?

Reply:
With HiveCatalog

Reply:
I see - it's because Hive catalog overrides this method and always gives back null. Just wasn't clear by looking at the base method itself

Reply:
Updated the comment for clarity.
Thanks for the review!